src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -24,6 +24,13 @@ tls [internal|<email>] | [<cert_file> <key_file>] {
 	ca_root   <pem_file>
 	dns       <provider_name> [<params...>]
 	on_demand
+	clients {
+		mode [request|require|verify_if_given|require_and_verify]
+		trusted_ca_certs        <trusted_ca_certs...>
+		trusted_ca_certs_file   <filename...>
+		trusted_leaf_certs      <trusted_leaf_certs...>
+		trusted_leaf_certs_file <filename...>
+	}
```

Comment:
The syntax has changed a little bit recently: https://github.com/caddyserver/caddy/pull/3335/files#diff-3235e86b75f6440c822f6d7d8f4beb61R65-R71